content/articles/getting-started-clickfunnels.md

```diff
@@ -71,13 +67,13 @@ Welcome to [DNSimple](/articles/dnsimple-services/)! We're here to make your tra
 ## Step 3: Accepting and setting up the domain in DNSimple
 
 <div class="section-steps" markdown="1">
-1. [Log into DNSimple](https://dnsimple.com/login). You'll see a <label>Get Started</label> screen. Select <label>Skip this and go to the dashboard</label>, found under the three boxes of options. We'll go over these other options in the [section below](#other-options-when-getting-started-with-a-new-account).
+1. [Log in to DNSimple](https://dnsimple.com/login). You'll see a <label>Get Started</label> screen. Select <label>Skip this and go to the dashboard</label>, found under the three boxes of options. We'll go over these other options in the [section below](#other-options-when-getting-started-with-a-new-account).
 1. Welcome to the [Dashboard](https://dnsimple.com/dashboard) – the homepage of your DNSimple account. There's a box to the right that says <label>Alerts</label>. Click <label>View</label> to see your pending domain push and accept it.
 1. <label>Push Requests</label> and your domain are listed below. Click <label>Accept</label> to the right of the domain.
```

Comment:
Is this under **Push Requests** or **Accepting a transfer**?

Reply:
Push Requests